config/mappings/mapping_parts/scored_criteria.py

```diff
@@ -19,14 +19,14 @@
                                 "field_id": "kxgWTy",
```

Comment:
https://github.com/communitiesuk/funding-service-design-assessment-store/blob/830090b373fe9f0ad135007ff540d04ab4e2f174/config/mappings/mapping_parts/scored_criteria.py#L354-L367

Should these also be refactored to use the same `heading`, `description`, `amount` pattern you've been using? I've done this on my local as I've needed it to test the UI side of things:

Note the "heading" doesn't actually have a question, because I don't think the table has headings for these, so it's a blank string.

```
diff --git a/config/mappings/mapping_parts/scored_criteria.py b/config/mappings/mapping_parts/scored_criteria.py
--- a/config/mappings/mapping_parts/scored_criteria.py
+++ b/config/mappings/mapping_parts/scored_criteria.py
                                 "field_id": "TzOokX",
                                 "form_name": "project-costs",
                                 "field_type": "multiInputField",
-                                "presentation_type": "list",
+                                "presentation_type": "heading",
+                                "question": "",
+                            },
+                            {
+                                "field_id": "TzOokX",
+                                "form_name": "project-costs",
+                                "field_type": "multiInputField",
+                                "presentation_type": "description",
                                 "question": "Income source",
                             },
                             {
+                                "field_id": "TzOokX",
+                                "form_name": "project-costs",
+                                "field_type": "multiInputField",
+                                "presentation_type": "amount",
+                                "question": "Amount",
+                            },
+                            {
+                                "field_id": "fbFeEx",
+                                "form_name": "project-costs",
+                                "field_type": "multiInputField",
+                                "presentation_type": "heading",
+                                "question": "",
+                            },
+                            {
                                 "field_id": "fbFeEx",
                                 "form_name": "project-costs",
                                 "field_type": "multiInputField",
-                                "presentation_type": "list",
+                                "presentation_type": "description",
                                 "question": "Running costs",
                             },
+                            {
+                                "field_id": "fbFeEx",
+                                "form_name": "project-costs",
+                                "field_type": "multiInputField",
+                                "presentation_type": "amount",
+                                "question": "Amount",
+                            },
                         ],
                     },
                 ],

```

Reply:
Yes Thomas, I am working on it 😄